packages/webpack-loader/src/extract-plugin.ts

```diff
@@ -108,6 +108,13 @@ export class CompiledExtractPlugin {
 
   constructor(options: CompiledExtractPluginOptions = {}) {
     this.#options = options;
+
+    // @ts-expect-error -- Make sure this config doesn't bleed in as it's passed through
+    if (options.classHashPrefix) {
+      throw new Error(
+        '`@compiled/webpack-loader.CompiledExtractPlugin` is mixing `extract: true` and `classHashPrefix` options, which will not supported and will result in bundle size bloat.'
+      );
+    }
```

Comment:
I've removed the config actually, so you can't do `CompiledExtractPlugin({ classHashPrefix: … })`, but kept the `@ts-expect-error` to document it in full (because it'd work without TS blocking you, I believe.